spec/decorators/table_sorters/hearings_sorter_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'court_data_adaptor'
+
+RSpec.describe TableSorters::HearingsSorter do
+  subject(:instance) { described_class.new(hearings, column, direction) }
+
+  include_context 'with multiple hearings to sort'
+
+  describe '#sorted_hearing_days' do
+    subject(:call) { instance.sorted_hearing_days(hearing1) }
+
+    let(:column) { 'provider' }
+    let(:direction) { 'asc' }
+    let(:expected_result) do
+      ['2021-01-19T10:45:00.000Z'.to_datetime, '2021-01-20T10:45:00.000Z'.to_datetime]
+    end
+
+    it 'sorts hearing by hearing_days asc' do
+      is_expected.to eql(expected_result)
+    end
+  end
+
+  describe '.for' do
+    subject { described_class.for(hearings, column, direction) }
+
+    context 'when column is provider and direction is asc' do
+      let(:column) { 'provider' }
+      let(:direction) { 'asc' }
+
+      it 'sorts hearing by hearing_days desc' do
+        is_expected.to be_instance_of(TableSorters::HearingsProviderSorter)
+      end
```

Comment:
This description looks like it should be empty/consistent with examples elow

```suggestion
      it {
        is_expected.to be_instance_of(TableSorters::HearingsProviderSorter)
      }
```

Reply:
have add a commit this this affect